apps/experiments/tests/test_models.py

```diff
@@ -22,24 +24,38 @@ def experiment_session():
     return ExperimentSessionFactory()
 
 
+@pytest.fixture()
+def _load_voice_data(db):
+    """Ensures that the voice data is loaded in the DB"""
+    data_dir = settings.BASE_DIR / "apps/experiments/migrations/preload_data"
+    voice_files = ["aws_voices.json", "azure_voices.json", "openai_voices.json"]
+    for voice_file in voice_files:
+        file_path = data_dir / voice_file
+        migration_utils.create_synthetic_voices_from_file(SyntheticVoice, file_path=file_path)
+
+
 class TestSyntheticVoice:
     @pytest.mark.django_db()
+    @pytest.mark.usefixtures("_load_voice_data")
```

Comment:
Did the [django_db_with_data](https://github.com/dimagi/open-chat-studio/blob/c6709572b4a46fe973db881713a13efc7ff95c41/apps/utils/pytest.py#L1) not work?

Reply:
It does, thanks